skyportal/models/telescope.py

```diff
@@ -312,6 +312,6 @@ def current_time(self):
 
         return {
             "is_night_astronomical": is_night_astronomical,
-            "next_twilight_morning_astronomical": morning,
```

Comment:
Can you confirm the below is the only place this naming is used?

Reply:
Thanks for the review Michael. I added that method yesterday and its only used in the telescope handler, map and obsplan queue  which I corrected as well if necessary. So that should be fine.